catalog/src/main/res/values/strings.xml

```diff
@@ -88,4 +90,9 @@
     <string name="component_menu_issue">Report an issue</string>
 
     <string name="icons_screen_search_helper">Search Spark Icon</string>
-    </resources>
+
+    <string name="spark_text_link_paragraph_example_"><annotation typography="body1" color="neutral">This is Adevinta </annotation><annotation typography="body1" color="main"><u><b>Privacy &amp; Policy</b></u></annotation><annotation typography="body1" color="neutral">also lots of extra information you maybe interested in or should I inform you about that extra information. you might be not knowing it</annotation></string>
```

Comment:
@kazaky I'm not sure why these have been renamed with trailing underscores 🤔

Reply:
There was a build error for duplicate res. I just renamed the String res and its fixed